demo/remote/azure/terraform/control/main.tf

```diff
@@ -0,0 +1,17 @@
+module "my_ip_address" {
+  source = "matti/resource/shell"
+
+  command = "curl https://ipinfo.io/ip"
+}
+
+module "hashistack_cluster" {
+  source = "../modules/azure-hashistack"
+
+  allowlist_ip = ["${module.my_ip_address.stdout}/32"]
```

Comment:
To test this with Azure fingerprinting, you can build Nomad from `master`, zip it and put somewhere public (like S3). Then add a new line here:

```hcl
nomad_binary = "https://...
```